app/controllers/application_controller/current_user.rb

```diff
@@ -12,14 +16,12 @@ def clear_current_user
     session[:userid]    = nil
     session[:group]     = nil
   end
-  protected :clear_current_user
```

Comment:
I used to have it as 'current_user = nil`. Thought this was a little more clear, but I'm on the fence. Need to sleep on it, but think you're right here..